build.sh

```diff
@@ -178,10 +178,10 @@ if buildAll || hasArg cudf; then
 
     cd ${REPODIR}/python/cudf
     if [[ ${INSTALL_TARGET} != "" ]]; then
-        python setup.py build_ext --inplace
+        env PARALLEL_LEVEL=${PARALLEL_LEVEL} python setup.py build_ext --inplace
```

Comment:
Any reason we can't just remove the `env` and make it:
```
PARALLEL_LEVEL=${PARALLEL_LEVEL} python setup.py build_ext --inplace
```

Reply:
`env` launches a new process which feels weird and unnecessary here

Reply:
Yeah that should be fine